.gitignore

```diff
@@ -2,7 +2,7 @@
 target/
 .scalafmt.conf
 .idea
-.swp
+*.swp
```

Comment:
Which part of our process produces .swp files?

Reply:
Me, using `vim` :)